4-fleetscope/modules/env_baseline/variables.tf

```diff
@@ -14,3 +14,29 @@
  * limitations under the License.
  */
 
+variable "env" {
```

Comment:
It appears that (currently) this variable isn't used?

Reply:
changed to use it in the scope_id